docs/navigation-configuration.md

```diff
@@ -58,6 +59,10 @@ window.Luigi.setConfig({
 
 ### Navigation parameters
 
+- **nodeAccessibilityResolver** allows you to define a permission checker function that gets executed on every Node. If it returns false, the Node and it's children are removed from the navigation structure. You may want to add custom values to Nodes to receive it in the **nodeAccessibilityResolver** function. See [angular sampleconfig.js](../core/examples/luigi-sample-angular/src/assets/sampleconfig.js) for our constraints example.
```

Comment:
* it's children -> `its children`
* it returns false -> ``it returns `false` ``
* preferably it would be great if you could replace the passive voice here with such a structure:``If it returns `false`, {subject} removes the Node and its children from the navigation structure.``
* You may want to add custom values to a Node to receive it in the **nodeAccessibilityResolver** function. ->  ` To receive a Node in the **nodeAccessibilityResolver** function, add custom values to it. `(I hope the meaning is kept?)